### PR TITLE
WIP: Hydrate .Command in common.proto

### PR DESCRIPTION
**Pull request by dio**
This change hydrates the Command message to let it be compiled in C++ env.

### CODE REVIEW

**Comment by dio** [comment]
Still in progress on checking up things in, since the compilation is still in failure...

**Comment by wu-sheng** [comment]
@dio Do you want me to merge this first?

**Comment by dio** [comment]
@wu-sheng not yet. I'll figure out from my side since I have to deal with this bazel thing...

File: service-mesh-probe/service-mesh.proto [hunk 1]
```diff
@@ -52,5 +52,14 @@ enum Protocol {
     gRPC = 1;
 }
 
+// In most cases, detect point should be `server` or `client`.
+// Even in service mesh, this means `server`/`client` side sidecar
+// `proxy` is reserved only.
+enum DetectPoint {
```
Comment (wu-sheng) [inline service-mesh-probe/service-mesh.proto]: @dio Please import `common/common.proto` to find this object. Such as

> import "common/common.proto";

**Comment by dio** [comment]
This is not necessary, my bad.

**Pull request closed without merging**